crates/partition-store/proto/dev/restate/storage/v1/domain.proto

```diff
@@ -118,12 +118,12 @@ message Source {
 message InvocationStatusV2 {
 
   enum Status {
+    reserved 6;
```

Comment:
Maybe add a comment that this value was used for an experimental feature that was removed.